pkg/flowaggregator/util/options/options.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2022 Antrea Authors
```

Comment:
why does this need to be in a `util` package"? Could it be `pkg/flowaggregator/options.go` or `pkg/flowaggregator/options/options.go`?

Reply:
It supposed to be `pkg/flowaggregator/options/options.go`
Fixed. Thanks.